src/aws-api-mcp-server/awslabs/aws_api_mcp_server/server.py

```diff
@@ -65,9 +69,35 @@
 log_file = log_dir / 'aws-api-mcp-server.log'
 logger.add(log_file, rotation='10 MB', retention='7 days')
 
+
+def setup_server_config():
```

Comment:
Let's extract this into a separate (sensible) file and not have a new floating function in server.py